client/pages/[state].js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import { useRouter } from 'next/router'
+import { regenCycle } from 'constants'
+
+import SharedLayout from "components/SharedLayout";
+import PlaceholderText from "components/mock/Placeholder";
+import Breadcrumbs from "components/common/Breadcrumbs";
+
+export default function State() {

```

Comment:
It's cool that this dynamically generates a page! :-O At some point we'll have to add a check that the state is valid and otherwise redirect to a 404 or something.

Reply:
Yep exactly - we'll pull a list of the valid states from the API, apparently NextJS can dynamically generate all the individual pages as a build process api call. I'm a little curious if it will automatically forward to 404 after generating the pages.

This is just a very small shell for now obviously.